<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DistBarWidget</class>
 <widget class="QWidget" name="DistBarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1,1">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_text">
     <property name="text">
      <string>lbl_text</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item row="0" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="lbl_min_value">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="lbl_min_value_2">
       <property name="text">
        <string>|</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="lbl_max_value">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2" alignment="Qt::AlignRight">
    <widget class="QLabel" name="lbl_value">
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QWidget" name="widget_bar" native="true"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
